@types/environment.d.ts

```diff
@@ -3,6 +3,7 @@ declare global {
     interface ProcessEnv {
       // @ts-expect-error  Nodejs process override
       readonly NODE_ENV: 'development' | 'production' | 'test' | 'build';
+      readonly JWT_SECRET: '@developer@NkowaOkwu@secret@key@';
```

Comment:
we wouldn't want to specify the actual value of `JWT_SECRET` - instead, we would want to say that `JWT_SECRET` can be any string
```suggestion
      readonly JWT_SECRET: string
```